Assets/i5 Toolkit for Unity/Runtime/Importers/Scripts/ObjImporter.cs

```diff
@@ -94,9 +93,9 @@ public async Task<GameObject> ImportAsync(string url)
                 // check that the referenced mtl library is already loaded; if not: load it
                 if (!MtlLibrary.LibraryLoaded(parseResult.LibraryPath))
                 {
-                    string mtlUri = UriUtils.RewriteFileUriPath(uri, parseResult.LibraryPath);
-                    string libraryName = System.IO.Path.GetFileNameWithoutExtension(uri.LocalPath);
-                    bool successful = await MtlLibrary.LoadLibraryAsyc(new Uri(mtlUri), libraryName);
+                    string mtlUri = System.IO.Path.ChangeExtension(path, ".mtl");

```

Comment:
This line changes the functionality and can lead to errors. The new version implies that the material file will always have the same name as the obj file and will be in the same directory. However, there are online repositories where the file is named differently or called differently. The original RewriteFileUriPath method takes the absolute path to the obj file and concatenates the relative library path.

The new version e.g. leads to errors when loading 3D model from https://people.sc.fsu.edu/~jburkardt/data/obj/obj.html.

Example: The file https://people.sc.fsu.edu/~jburkardt/data/obj/airboat.obj has its mtl library at ./vp.mtl. When trying to import it, it raises the follow error message:

![grafik](https://user-images.githubusercontent.com/29382867/123926418-238ca200-d98c-11eb-8f5b-e4cf42aa8db5.png)

Maybe something similar to this works for both URLs and local file paths?
`string absolutePath = Path.GetFullPath(baseDirectory + relativePath);`